src/base/b-list/b-list.ts

```diff
@@ -703,6 +687,13 @@ export default class bList extends iData implements iVisible, iWidth, iItems {
 		}
 	}
 
+	/**
+	 * Returns true if the component is used as tab list
```

Comment:
Геттер имеет интерфейс свойства, поэтому глагол в описании не уместен.

```suggestion
	 * True if the component is used as a tablist
```